app/models/manageiq/providers/vmware/network_manager/refresh_parser.rb

```diff
@@ -11,14 +11,14 @@ def initialize(ems, options = nil)
       @data_index = {}
       @inv        = Hash.new { |h, k| h[k] = [] }
       @org        = @connection.organizations.first
+      @network_name_mapping = {}
     end
 
     def ems_inv_to_hashes
-      log_header = "MIQ(#{self.class.name}.#{__method__}) Collecting data for EMS name: [#{@ems.name}] id: [#{@ems.id}]"
```

Comment:
Why are you removing this, but not the next  line?

Reply:
Ok, I see now that `log_header` is now a function that you use in various places. Please see my comment [there](https://github.com/ManageIQ/manageiq-providers-vmware/pull/161/files#diff-b90e0cdaee1f8e3c7c23889a3fc00f93R32).